### PR TITLE
refactor(ScoreSummary): add optional description text

### PR DESCRIPTION
**Pull request by natalieweesh**
### Overview

add optional description prop to ScoreSummary component

Screen Shot 2022-11-28 at 11 15 00 AM

Screen Shot 2022-11-28 at 11 15 06 AM

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: [EGG-2949]
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

[EGG-2949]: https://codecademy.atlassian.net/browse/EGG-2949?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.236-alpha.fc55f7.0<br/>@codecademy/gamut-labs@33.8.5-alpha.fc55f7.0<br/>@codecademy/styleguide@62.5.5-alpha.fc55f7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6384e1445737091eb9176f46--gamut-preview.netlify.app>https://6384e1445737091eb9176f46--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6384e1445737091eb9176f46>Deploy Logs</a></p>

File: packages/gamut-labs/src/ScoreSummary/index.tsx [hunk 1]
```diff
@@ -184,6 +186,17 @@ export const ScoreSummary: React.FC<ScoreSummaryProps> = ({
             columnLayout={columnLayout}
           />
         </Box>
+        {description && (
+          <Box
+            p={16}
+            pb={layout === 'row' ? 0 : 16}
+            border={layout === 'row' ? 'none' : 1}
+            borderTop="none"
+            textAlign={layout === 'row' ? 'center' : 'left'}
```
Comment (sylvanasantos) [inline packages/gamut-labs/src/ScoreSummary/index.tsx:195]: super nit: you use this `layout === 'row'` check multiple times throughout this file, maybe you could save it in a `isLayoutRow` variable?

**Review by sylvanasantos — APPROVED** [review]
💅

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.236-alpha.7a6952.0<br/>@codecademy/gamut-labs@33.8.5-alpha.7a6952.0<br/>@codecademy/styleguide@62.5.5-alpha.7a6952.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6384fad11f5659300d87901a--gamut-preview.netlify.app>https://6384fad11f5659300d87901a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6384fad11f5659300d87901a>Deploy Logs</a></p>

**Review by BandanaKM — APPROVED** [review]
looks great!

**Pull request merged**